src/test/rpc/AccountInfo_test.cpp

```diff
@@ -308,22 +370,240 @@ class AccountInfo_test : public beast::unit_test::suite
             }
-            BEAST_EXPECT(info.isMember(jss::jsonrpc) && info[jss::jsonrpc] == "2.0");
-            BEAST_EXPECT(info.isMember(jss::ripplerpc) && info[jss::ripplerpc] == "2.0");
+            BEAST_EXPECT(
+                info.isMember(jss::jsonrpc) && info[jss::jsonrpc] == "2.0");
+            BEAST_EXPECT(
+                info.isMember(jss::ripplerpc) && info[jss::ripplerpc] == "2.0");
             BEAST_EXPECT(info.isMember(jss::id) && info[jss::id] == 6);
         }
     }
 
-    void run() override
+    // gRPC stuff
+    class GetAccountInfoClient : public GRPCTestClientBase
+    {
+    public:
+        rpc::v1::GetAccountInfoRequest request;
+        rpc::v1::GetAccountInfoResponse reply;
+
+        explicit GetAccountInfoClient(std::string const& port)
+            : GRPCTestClientBase(port)
+        {
+        }
+
+        void
+        GetAccountInfo()
+        {
+            status = stub_->GetAccountInfo(&context, request, &reply);
+        }
+    };
+
+    void
+    testSimpleGrpc()
+    {
+        testcase("gRPC simple");
+
+        using namespace jtx;
+        std::unique_ptr<Config> config = envconfig(addGrpcConfig);
+        std::string grpcPort = *(*config)["port_grpc"].get<std::string>("port");
+        Env env(*this, std::move(config));
+        Account const alice{"alice"};
+        env.fund(drops(1000 * 1000 * 1000), alice);
+
+        {
+            // most simple case
+            GetAccountInfoClient client(grpcPort);
+            client.request.mutable_account()->set_address(alice.human());
+            client.GetAccountInfo();
+            if (!BEAST_EXPECT(client.status.ok()))
+            {
+                std::cout << client.reply.DebugString() << std::endl;
+                return;
+            }
+            BEAST_EXPECT(
+                client.reply.account_data().account().address() ==
+                alice.human());
+        }
+        {
+            GetAccountInfoClient client(grpcPort);
+            client.request.mutable_account()->set_address(alice.human());
+            client.request.set_queue(true);
+            client.request.mutable_ledger()->set_sequence(3);
+            client.GetAccountInfo();
+            if (!BEAST_EXPECT(client.status.ok()))
+                return;
+            BEAST_EXPECT(
+                client.reply.account_data().balance().drops() ==
+                1000 * 1000 * 1000);
+            BEAST_EXPECT(
+                client.reply.account_data().account().address() ==
+                alice.human());
+            BEAST_EXPECT(
+                client.reply.account_data().sequence() == env.seq(alice));
+            BEAST_EXPECT(client.reply.queue_data().txn_count() == 0);
+        }
+    }
+
+    void
+    testErrorsGrpc()
+    {
+        testcase("gRPC errors");
+
+        using namespace jtx;
+        std::unique_ptr<Config> config = envconfig(addGrpcConfig);
+        std::string grpcPort = *(*config)["port_grpc"].get<std::string>("port");
```

Comment:
you could just access `env.app().config()["port_grpc"].get<std::string>("port")` after constructing the `env` I think.

Reply:
I see. I don't think that's significantly better than what I'm doing here, so I'd rather just leave it, unless you strongly object